docs: Adjust documentation, fix typos, update locale.

FayeDel opened the pull request:
## About

This pull request fixes missing references to documentation and adjusts for the new RTD link, including the quickstart README link pointing.
This is NOT meant to replace the Furo PR, this is merely a short term hot fix until the documentation rewrite.

As of 26681ee, the localisation has been updated to adjust for the new changes as well.

## Checklist

- [X] I've ran `pre-commit` to format and lint the change(s) made.
- [X] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [X] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent): #789 
- [X] I've made this pull request for/as: (check all that apply)
  - [X] Documentation
  - [ ] Breaking change
  - [X] New feature/enhancement
  - [X] Bugfix

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

i0bs committed:
chore!: pre-release for 4.1.1-rc.1 (#703)

chore!: pre-release for 4.1.1-rc.1

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

i0bs committed:
ADMIN/revert: remove unwanted file.

This file was actually cloned outside of the client directory in a PR that had not appropriately adjusted to the new architectural layout of the library.

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

FayeDel committed:
Merge pull request #777 from interactions-py/unstable

chore!: release 4.2.0

FayeDel committed:
chore: Update requirements.txt

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

FayeDel committed:
Merge branch 'interactions-py:unstable' into unstable

FayeDel committed:
Merge pull request #785 from interactions-py/unstable

chore!: release 4.2.1

FayeDel committed:
chore: Version bump.

FayeDel committed:
docs: Adjust documentation linking, fix typos.

sillyElvy commented on docs/migration.rst:
```diff
@@ -61,4 +61,4 @@ portal and add the intent to your current intents when connecting:
     from interactions import Client, Intents
 
-    bot = Client("TOKEN", intents=Intents.DEFAULT | Intents.MEESAGE_CONTENT)
+    bot = Client("TOKEN", intents=Intents.DEFAULT | Intents.MESSAGE_CONTENT)
```
It's guild message content iirc

FayeDel committed:
docs: Fix message intent reference.

i0bs committed:
chore/ADMIN: force version onto old pypi

This is an executive decision made between Delta and I. We will be forcing a version 4.0 push onto our old branch to help transition old users using v3 to our new project. This does not necessarily mean this branch will now be receiving updates along with the rest. The goal is to make our current branch used as much as possible.

i0bs committed:
chore/ADMIN: fix content type

i0bs committed:
fix/ADMIN: correct improper description

i0bs committed:
revert/ADMIN: change back to original

sillyElvy commented:
I just checked the docs build, the client page is not built correctly, may be worth having another look at it

FayeDel committed:
docs: Fix Client reference.

FayeDel committed:
docs: Remove Modal beta error tag, fix grammar typo

FayeDel committed:
docs: Include private objects by default.

FayeDel committed:
docs: Rebuild .mo/.po localisation for locales.

This may wipe current translation, however the lib docs are different now than the last build (~25/9/2021)

pre-commit-ci[bot] committed:
ci: correct from checks.

FayeDel committed:
docs: Refactor language build default for PR review, remove :inherit: tag.

FayeDel committed:
Merge remote-tracking branch 'origin/unstable-docs-repair' into unstable-docs-repair

FayeDel committed:
docs: Include rest of locale (ext, context, etc).

This was cut off from some random PyCharm setting.

pre-commit-ci[bot] committed:
ci: correct from checks.

FayeDel committed:
docs: Fix codeblocks in some areas, fix gateway documentation

FayeDel committed:
Merge branch 'interactions-py:stable' into unstable-docs-repair

FayeDel committed:
docs: Implement barebones better search capability

This seems to, according to docs, only work when the server's live. But on a PR view it acts as if this commit doesn't do anything.

FayeDel committed:
docs: Rebuild locale.

pre-commit-ci[bot] committed:
ci: correct from checks.

FayeDel committed:
docs: Correct models.command.po locale.

FayeDel committed:
docs: Adjust note/warning label spacing on component.py

FayeDel committed:
docs: Adjust return doc spacing, rebuild locale.

FayeDel committed:
Merge branch 'unstable' into unstable-docs-repair

# Conflicts:
#	interactions/api/models/user.py
#	interactions/base.py
#	interactions/client/models/command.py

FayeDel committed:
chore: Rebase from unstable, convert to attrs, rebuild locale.
